new-pipeline/src/dashboards.js

```diff
@@ -1048,8 +1048,14 @@ function updateOSs() {
 
 function getDescriptionWithLink(metric, channel, description) {
   var metricUrl = "https://georgf.github.io/fx-data-explorer/index.html?search=" + metric + "&searchtype=in_name&optout=false&channel=" + channel + "&constraint=is_in&version=any"
-  var imgUrl = "https://upload.wikimedia.org/wikipedia/commons/6/64/Icon_External_Link.png"
   var descr = description === null ? metric : description
 
-  return " <a href=\"" + metricUrl + "\" target=\"_blank\">" + descr + "<img src=\"" + imgUrl + "\"/></a>"
+  var div = document.createElement('div');
+  div.classList.add("text-center");
+  div.innerHTML =  descr + " <a href=\"" + metricUrl + "\" target=\"_blank\">" +
+                   "<i class=\"btn btn-outline-primary fa fa-info-circle\" " +
+                   "style='float: right; color: black' aria-hidden=\"true\">  " +
+                   "More Details </i></a>"
```

Comment:
Two things here from the previous feedback that were not addressed:
- This still uses the `<i>` element.
- This still uses `float: right;`

Reply:
I had already mentioned previously that the icon of more details uses the <i> tag so only if you don't want the icon I shall remove it

I had posted screenshots related to why as to float : right spoils the view , basically because then it just sticks right next to where the description ends . However I shall remove that if that is what is really needed